### ISSUE TITLE
Offer all nn.Module subclasses in common package to import

### ISSUE THREAD

**Issue by JoanFM** [kornia/kornia#988]
## 🚀 Feature
It would be cool to have every `class` that inherits from `nn.Module` in `kornia` to be accessible via `kornia` parent `import` (or through some common package)

What I mean would be:

enable
```
from kornia import Normalize
```
instead of

```
from kornia.augmentation.Normalize
```

## Motivation

I would like to `wrap` a class around `kornia` transformations allowing the user to describe `arbitrary` sequence of transformations.

It would look like this:

```python

import kornia
class KorniaWrapper()

        def __init__(
            self,
            transforms: List[Union[str, Dict[str, Dict]]] = ['Normalize', {'mean': 0.485, 'std': 0.01],
            *args,
            **kwargs
        ):
                transformations = []
                for tr_name, tr_kwargs in transforms
                     transform = getattr(kornia, tr_name)(**tr_kwargs)
                     ransformations.append(transform)

              self.transform = nn.Sequential(**transformations)

       def process(images):
            return self.transform(images)
      
```

**Comment by edgarriba** [comment]
@JoanFM thanks for asking about this. We had a discussion already with @shijianjian and @cceyda  about this topic.
IMO, exposing everything at level could go against the internal package organization and lead to confusion since we have already large set of functionalities. Is there any workaround that could be done through filtering python globals ? let's keep this discussion to fit in v0.6

**Comment by shijianjian** [comment]
It is discussed in #520. The basic idea is to have a Kornia.nn module to contain all those nn.module implementations. To separate the functional and modulized implementations.

I like the idea of having a wrapper to compose a transformation sequence out of a JSON. Probably expose all the modules in the Kornia.nn level? @edgarriba

**Comment by edgarriba** [comment]
Let’s do that. It’s pretty much how is pytorch organised, so makes sense. But for sure augmentations have to be kept as separated modules as we discussed for the nature of the ops since include random sampling.

**Comment by edgarriba** [comment]
@JoanFM to make it smooth, could you list some ops that you will like to have at first ?

**Comment by JoanFM** [comment]
Hey @edgarriba ,

The most interesting ones are definitely `Normalize` and some `cropping`, and some common preparing transformations for inference purposes?

Also `resize` would be interesting but I guess with `resize` operations there may be a problem with handling resize operations with batches of more than one image

**Comment by edgarriba** [comment]
@JoanFM we provide now a `Lamba` functionality to wrap functionals in terms of a nn module. You could do as follows:
```python 

f = Lambda(lambda x: kornia.color.rgb_to_grayscale(x))

x = torch.rand(1,3,224,224)
y = f(x)  # 1x3x224x224

```
